ext/opcache/tests/blacklist.phpt

```diff
@@ -8,7 +8,14 @@ opcache.file_update_protection=0
 opcache.file_cache_only=0
 --SKIPIF--
 <?php require_once('skipif.inc'); ?>
-<?php if (substr(PHP_OS, 0, 3) == 'WIN') { die('skip not for Windows'); } ?>
+<?php
+if (substr(PHP_OS, 0, 3) == 'WIN') {
+    die('skip not for Windows');
+}
+/* On macOS, `/tmp` is an alias to `/private/tmp` .
+ * So, we should write `%S/tmp/path` as `/tmp/path` not for Windows.
```

Comment:
The "not for Windows" part of this sentence looks odd.

Reply:
I fixed.